src/Matmar10/Money/Entity/Money.php

```diff
@@ -79,17 +112,17 @@ public function getCurrency()
 
     public function getScale()
     {
-        return $this->scale;
+        return $this->scale ?: $this->scale = bcpow(10, $this->currency->getPrecision(), 0);
```

Comment:
this is a trade of to use this class as embedded property. there is no event or callback to set the scale after load this entity from database as embedded style :(